### PR TITLE
docs: Livestream Tutorial

### PR DESCRIPTION
**Pull request by oliverlaz**
### Overview

A simplified tutorial for the Livestream sample app.

<details><summary>Details</summary>
<p>

localhost_3000_video_docs_react_tutorials_livestream_

</p>
</details>

### CODE REVIEW

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 1]
```diff
@@ -1,6 +1,339 @@
 ---
 title: Livestream Tutorial
-description: How to build a livestream experience using Stream's video SDKs
+description: How to build a live stream experience using Stream's React Video SDKs
 ---
 
-TODO
+## Quickstart
+In this quickstart, we'll build a live streaming experience that's similar to Twitch.
```
Comment (DaemonLoki) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:7]:
```suggestion
In this quickstart, we'll build a live-streaming experience that is similar to Twitch.
```

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 2]
```diff
@@ -1,6 +1,339 @@
...
+- The live stream will run on Stream's Edge network of servers around the world
+- You can use ultra-low latency (WebRTC) based live streaming or HLS (slight delay, but better buffering)
+- We'll show how to publish to the live stream from your browser
+- Stream enables you to scale to millions of viewers
+- You can have multiple active participants in the live stream
+
+The app that we are going to build would work in two modes: Host (presenter) and Viewer (audience).
+
+The host will be able to publish the live stream and the viewers will be able to watch the live stream.
+If you want to see a fully-fledged example, check out this [sample app.](https://github.com/GetStream/stream-video-js/tree/main/sample-apps/react/livestream-app)
```
Comment (DaemonLoki) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:18]:
```suggestion
If you want to see a full-blown example, check out this [sample app.](https://github.com/GetStream/stream-video-js/tree/main/sample-apps/react/livestream-app)
```

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 3]
```diff
@@ -1,6 +1,339 @@
...
-description: How to build a livestream experience using Stream's video SDKs
+description: How to build a live stream experience using Stream's React Video SDKs
 ---
 
-TODO
+## Quickstart
+In this quickstart, we'll build a live streaming experience that's similar to Twitch.
+
+- The live stream will run on Stream's Edge network of servers around the world
+- You can use ultra-low latency (WebRTC) based live streaming or HLS (slight delay, but better buffering)
```
Comment (DaemonLoki) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:10]:
```suggestion
- We can use ultra-low latency ([WebRTC](https://webrtc.org/)) based live streaming or [HLS](https://en.wikipedia.org/wiki/HTTP_Live_Streaming) (slight delay, but better buffering)
```

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 4]
```diff
@@ -1,6 +1,339 @@
...
+- We'll show how to publish to the live stream from your browser
```
Comment (DaemonLoki) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:11]:
```suggestion
- We will go through the process of publishing the live stream from the browser
```

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 5]
```diff
@@ -1,6 +1,339 @@
...
+- Stream enables you to scale to millions of viewers
```
Comment (DaemonLoki) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:12]: We could explain a bit more that this is done automagically and the user doesn't do anything. Might be misleading though, because costs will obv. increase....not sure if it makes sense.

```suggestion
- Stream enables us to automatically scale to millions of viewers (no actions required from your side)
```

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 6]
```diff
@@ -1,6 +1,339 @@
...
+- You can have multiple active participants in the live stream
```
Comment (DaemonLoki) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:13]: I think the switch between "we can" and "you will" is a bit confusing to read. So I switched this to use we in all cases.

```suggestion
- We even show how to have multiple active participants in the live stream
```

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 7]
```diff
@@ -1,6 +1,339 @@
...
+
+The app that we are going to build would work in two modes: Host (presenter) and Viewer (audience).
```
Comment (DaemonLoki) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:15]:
```suggestion
The app that we are going to build will work in two modes: Host (presenter) and Viewer (audience).
```

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 8]
```diff
@@ -1,6 +1,339 @@
...
+
+## Step 1 - Preparations
+
+Before going into the code, let's make sure we have everything we need.
+
+- Add Stream's React Video SDK to your project:
+  - `yarn install @stream-io/video-react-sdk` or
+  - `npm install @stream-io/video-react-sdk`
+- Setup a Stream account at [GetStream's Dashboard](https://dashboard.getstream.io) or use your existing one
+  - We will need your API key and secret
```
Comment (DaemonLoki) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:30]: That sounds a bit like: GIVE US YOUR DATA!!!1!!11!

```suggestion
  - We will need an API key and secret to run the project
```

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 9]
```diff
@@ -1,6 +1,339 @@
...
+
+## Step 2 - Setup the host mode
+
+Hosts should be communicating with each other in real-time. For that purpose, we would use the standard Stream Video Call component, based on WebRTC.
```
Comment (DaemonLoki) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:34]: Do you think it makes sense to use *should* and *would* here? 

It sounds a bit theoretical whereas we're indeed creating a real application where everything *WILL* work

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 10]
```diff
@@ -1,6 +1,339 @@
...
+
+:::note
+In this tutorial, we will skip the details on how a `StreamVideoClient` can be set up. You can read more about it on the following page:
+[Authenticating your Users](../../guides/01-authentication/)
+:::
```
Comment (DaemonLoki) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:39]: Bringing some color into play and even linking to other content pages for viewers to go to...Oliver, you are a real Pro! 🚀👑

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 11]
```diff
@@ -1,6 +1,339 @@
...
+
+In the snippet below, we create a connection to the Stream Edge server for the live stream host, we join the call and we publish our audio and video to the live stream.
```
Comment (DaemonLoki) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:41]: I like this explanation a lot. Clear, precise, and to the point. Awesome!

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 12]
```diff
@@ -1,6 +1,339 @@
...
+In this example, we use the predefined `<PaginatedGridLayout>` for the live stream.
+
+In case you want to have a more custom layout, you can build your own. See [Building Custom Layouts.](../ui-cookbook/participant-list/)
```
Comment (DaemonLoki) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:44]:
```suggestion
In case you want to have a more custom layout, you can build your own. See our article about [Building Custom Layouts.](../ui-cookbook/participant-list/)
```

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 13]
```diff
@@ -1,6 +1,339 @@
...
+
+Hosts should be communicating with each other in real-time. For that purpose, we would use the standard Stream Video Call component, based on WebRTC.
+
+:::note
+In this tutorial, we will skip the details on how a `StreamVideoClient` can be set up. You can read more about it on the following page:
+[Authenticating your Users](../../guides/01-authentication/)
+:::
+
+In the snippet below, we create a connection to the Stream Edge server for the live stream host, we join the call and we publish our audio and video to the live stream.
+In this example, we use the predefined `<PaginatedGridLayout>` for the live stream.
```
Comment (DaemonLoki) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:42]: Do we have more info about the existing layouts somewhere? Would be a chance to link those.

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 14]
```diff
@@ -1,6 +1,339 @@
...
+
+In case you want to have a more custom layout, you can build your own. See [Building Custom Layouts.](../ui-cookbook/participant-list/)
+
+
+```tsx
```
Comment (DaemonLoki) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:47]: I am in no way able to judge the code itself. I feel like it's very long though and might be hard to follow in one bit. Do we have any chances of breaking things up a bit to not make it seem so long?

If not and it becomes unclear or harder to follow, then just keep it as is!

Very naive idea:
We could break it up and extract the `BackstageUI` component into a separate code block, quickly explaining what it does and what it is for...

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 15]
```diff
@@ -1,6 +1,339 @@
...
+## Step 3 - Enable the live stream for the viewers
+
+For simplicity, we are going to implement an additional button in the `BackstageControls` component that will enable the live stream for the viewers.
+
+The button should be able to toggle the live stream on and off. For that purpose, we are going to use two utility hooks from the SDK: 
+- [`useCall`](../04-call-engine/hooks-and-contexts.md#usecall) and
+- [`useIsCallBroadcastingInProgress`](../04-call-engine/hooks-and-contexts.md#useiscallbroadcastinginprogress)
+
+
+The `useCall` hook provides you with the current call object. This is useful when you want to access the call object from a component that is deep in the component tree with the `<StreamMeeting />` component as a predecessor.
```
Comment (DaemonLoki) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:147]: Let's keep it as "we" all the way through, switching between it and "you" is a bit confusing I think.

```suggestion
The `useCall` hook provides us with the current call object. This is useful when we want to access the call object from a component that is deep in the component tree with the `<StreamMeeting />` component as a predecessor.
```

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 16]
```diff
@@ -1,6 +1,339 @@
...
+
+On the other hand, the `useIsCallBroadcastingInProgress` hook provides you with the current broadcasting state of the call. This is useful when you want to show the current state of the live stream or build custom UI components or flows based on this call state.
```
Comment (DaemonLoki) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:149]:
```suggestion
On the other hand, the `useIsCallBroadcastingInProgress` hook gives us the current broadcasting state of the call. This is helpful when we want to show the current state of the live stream or build custom UI components or flows based on this call state.
```

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 17]
```diff
@@ -1,6 +1,339 @@
...
+  )
+};
+
+```
+
+:::info
+Starting and stopping a live stream for the viewers is an operation that takes a few seconds to complete.
+It is a good practice to show a loading indicator while the operation is in progress.
+
+There are two ways how you can do that:
```
Comment (DaemonLoki) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:180]:
```suggestion
There are two ways how we can do that:
```

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 18]
```diff
@@ -1,6 +1,339 @@
...
+### WebRTC-based live streaming
+
+For the WebRTC, we are going to use the same `<StreamMeeting />` component as in the host mode. The only difference is that we are going to use an anonymous user for the viewer.
+
+:::caution
+Anonymous users can't establish a WebSocket connection to our backend, therefore they are unable to receive any events from the call.
+This means that the viewer won't know when the call starts or stops broadcasting.
+
+To circumvent this limitation, the Viewer app should be able to be notified by the Host app or some other backend service when the call starts or stops broadcasting.
+Alternatively, you can keep polling the backend for the current state of the call. See [Querying Calls.](../02-guides/06-querying-calls.mdx)
```
Comment (DaemonLoki) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:202]:
```suggestion
Alternatively, you can keep polling the backend for the current state of the call. For more details on this, see [Querying Calls.](../02-guides/06-querying-calls.mdx)
```

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 19]
```diff
@@ -1,6 +1,339 @@
...
+      <video autoPlay playsInline ref={videoRef} onPlay={() => setIsPlaying(true)} />
+      
+      {!isPlaying && <LoadingIndicator />}
+      <ViewerControls />
+    </>
+  );
+};
+```
+
+Having this done, we should have a fully functional live streaming experience for the viewers based on HLS.
```
Comment (DaemonLoki) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:332]: Let's be bold here, we SHOULDN'T have it but in fact WE REALLY DO!!!💪🏻

```suggestion
Having done this, we now have a fully functional live-streaming experience for the viewers based on HLS.
```

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 20]
```diff
@@ -1,6 +1,339 @@
...
+
+:::tip
+For simplicity, a few things are omitted from the code snippet above. For the full-blown and working example,
+check this Livestreaming sample app:
+- [Livestream App source code](https://github.com/GetStream/stream-video-js/tree/main/sample-apps/react/livestream-app)
+- [Livestream App live preview](https://video-react-livestream-app.vercel.app/)
+:::
```
Comment (DaemonLoki) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:339]: Even suggesting other content to users. Hat tip, Oliver!

**Review by DaemonLoki — COMMENTED** [review]
Although the amount of my comments wouldn't naturally suggest it, I really like the article and there are only minor things I found. Awesome job!

**Review by szuperaz — APPROVED** [review]

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx (hunk 9 above)
Comment (MartinCupela) [reply packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:34]: I would change `Stream Video Call component` to a real component name.

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx (hunk 12 above)
Comment (MartinCupela) [reply packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:44]:
```suggestion
If you prefer a more customized layout, you can build your own. See our article about [Building Custom Layouts.](../ui-cookbook/participant-list/)
```

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 21]
```diff
@@ -1,6 +1,339 @@
...
+  const { publishVideoStream, publishAudioStream } = useMediaDevices();
+  useEffect(() => {
+    if (!call) return;
+    if (callState === CallingState.JOINED) {
+      // once the call is joined, publish the host's audio and video
+      publishVideoStream();
+      publishAudioStream();
+    }
+  }, [call, callState, publishAudioStream, publishVideoStream]);
+  return (
```
Comment (MartinCupela) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:105]:
```suggestion

  return (
```

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 22]
```diff
@@ -1,6 +1,339 @@
...
+  )
+}
+
+const BackstageUI = () => {
+  const call = useCall();
+  const callState = useCallCallingState();
+  // useMediaDevices() hook provides you quick access to the Browser's MediaDevices API
+  // and a few helper methods to manage your audio and video streams.
+  const { publishVideoStream, publishAudioStream } = useMediaDevices();
+  useEffect(() => {
```
Comment (MartinCupela) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:97]:
```suggestion

  useEffect(() => {
```

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx (hunk 2 above)
Comment (MartinCupela) [reply packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:18]: I am thinking, that with so many apps we provide in our tutorials, we could introduce an element / banner that would indicate clearly that this is the accompanying sample app / source code. WDYT? Speaking about being more visual and make it easier for readers to find it / determine, that accompanying src code exists.

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx (hunk 5 above)
Comment (oliverlaz) [reply packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx]: I think this is an implicit fact. I'll take your suggestion here :)

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx (hunk 7 above)
Comment (MartinCupela) [reply packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:15]: It occurred to me we could provide a description about how the app will work, e.g.:

In this app the host will initiate the livestream meanwhile the viewers are waiting for the livestream to start...

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx (hunk 2 above)
Comment (oliverlaz) [reply packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx]: @MartinCupela, I think this is a good idea. I'm sure other SDKs can benefit from such elements as they also provide sample apps like ours.
I'm not sure how much effort would be required to develop such a component, therefore, I'd leave it out for better days :)

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx (hunk 9 above)
Comment (oliverlaz) [reply packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:34]: Agreed, I'll update accordingly.

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx (hunk 13 above)
Comment (oliverlaz) [reply packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:42]: We should dedicate a chapter for them in `Video UI components`.

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 23]
```diff
@@ -1,6 +1,339 @@
...
+};
+
+```
+
+:::info
+Starting and stopping a live stream for the viewers is an operation that takes a few seconds to complete.
+It is a good practice to show a loading indicator while the operation is in progress.
+
+There are two ways how you can do that:
+- have a `useEffect` that would run whenever the we have initiated a change in the broadcasting state of the call until the value of `isBroadcasting` toggles.
```
Comment (MartinCupela) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:181]:
```suggestion
- have a `useEffect` that would run whenever we have initiated a change in the broadcasting state of the call until the value of `isBroadcasting` toggles.
```

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx (hunk 14 above)
Comment (oliverlaz) [reply packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:47]: @DaemonLoki I couldn't make the snippet shorter than this without losing important information. 

@MartinCupela I intentionally excluded a screenshot, as this tutorial doesn't build a UI, instead, it shows how a UI can be built.
At the end of the tutorial, the reader can see a live preview of our sample app, and play with it.

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx (hunk 19 above)
Comment (oliverlaz) [reply packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx]: Indeed! Fair point :)

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 24]
```diff
@@ -1,6 +1,341 @@
...
+        },
+        sort: [{ field: 'cid', direction: 1 }],
+      })
+      .then(({ calls }) => {
+        const [call] = calls;
+        if (call) {
+          setActiveCall(call);
+        }
+      });
+  }, [client, callId]);
```
Comment (MartinCupela) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:243]: It is not entirely clear to me, from where the `callId` comes. That could happen also to other readers.

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 25]
```diff
@@ -1,6 +1,341 @@
...
+  // pseudo-code for simplicity, see the previous example for the full code
+  const client = useCreateStreamVideoClient({...});
+  const [activeCall] = client.queryCalls({...});
+  return (
+    <StreamVideo client={client}>
+      <StreamMeeting 
+        call={activeCall} 
+        // we don't want to join this call, as that would switch to WebRTC-based live streaming.
+        // we are going to use HLS.js to play the live stream
+        autoJoin={false}
```
Comment (MartinCupela) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:298]:
```suggestion
// highlight-next-line
        autoJoin={false}
```

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 26]
```diff
@@ -1,6 +1,341 @@
...
+      >
+        <HLSLivestreamUI />
```
Comment (MartinCupela) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:300]:
```suggestion
// highlight-next-line
        <HLSLivestreamUI />
```

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx [hunk 27]
```diff
@@ -1,6 +1,341 @@
...
+      // show a loading indicator while the call is joining
+      {!isJoined && <LoadingIndicator />}
+      <ViewerControls />
+    </>
+  );
+};
+
+```
+
+### HLS-based live streaming
```
Comment (MartinCupela) [inline packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:272]: As the HLS example uses new hook `useCallMetadata` I would mention it and add a link to the generated docs.

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx (hunk 14 above)
Comment (MartinCupela) [reply packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:47]: I propose to add the links to the preview also at the beginning. I think it can help the understanding what is then read, because the reader can associate what was seen and experienced with the actual code. At least for me it always worked better.

Comment (oliverlaz) [reply packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:47]: The link to the source code is also available on the top of the tutorial but it isn't made that highlighted as the links at the end.

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx (hunk 9 above)
Comment (oliverlaz) [reply packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx]: Updated.

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx (hunk 24 above)
Comment (oliverlaz) [reply packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:254]: I've added more information.

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx (hunk 27 above)
Comment (oliverlaz) [reply packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:280]: Added.

File: packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx (hunk 14 above)
Comment (MartinCupela) [reply packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:48]: What if the Tip banner from the bottom was moved to the top?

Comment (oliverlaz) [reply packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx:48]: I'm not sure if we want to promote the App first. If we drag the readers to the app first, they might not know what they are looking at.
The guide shows an approach to how a lives stream app can be built. Then we provide a sample implementation.

@DaemonLoki, what is your take on this? What does more sense?
